null_safety_examples/basics/lib/main.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 The Flutter team. All rights reserved.
```

Comment:
If this is a new file, shouldn't the copyright be 2021?

Reply:
Done.